src/System.Private.CoreLib/src/System/Diagnostics/Eventing/DotNETRuntimeEventSource.cs

```diff
@@ -1,15 +1,20 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
+
 namespace System.Diagnostics.Tracing
 {
 #if FEATURE_PERFTRACING
     /// <summary>
     /// RuntimeEventSource is an EventSource that represents the ETW/EventPipe events emitted by the native runtime.
     /// Most of RuntimeEventSource is auto-generated by scripts/genRuntimeEventSources.py based on the contents of the Microsoft-Windows-DotNETRuntime provider.
     /// </summary>
+    [EventSource(Guid = "5E5BB766-BBFC-5662-0548-1D44FAD9BB56", Name = "Microsoft-Windows-DotNETRuntime")]
     internal sealed partial class RuntimeEventSource : EventSource
     {
+        // The RuntimeEventSource GUID is {5e5bb766-bbfc-5662-0548-1d44fad9bb56}
```

Comment:
Nit: This comment is not very useful. The GUID is is pretty clear from the attribute two lines above.

Reply:
Ok, I see that this is pattern from other CoreLib EventSources.